### ISSUE TITLE
Module.setFeatures doesn't work with TypeScript

### ISSUE THREAD

**Issue by cician** [AssemblyScript/binaryen.js#34]
Hi, I was playing with binaryen.js and stumbled upon a problem when trying to enable the Multivalue feature. After a few hours of trial end error I found that values of the ```Features``` enum assumes different values when compiled with Typescript proper compared to vanilla javascript or when using TypeScript with Parcel (which uses only Babel by default).

The following code outputs 10 when compiled with TypeScript and 512 (correct) from JavaScript.
```
const binaryen = require("binaryen");
console.log(binaryen.Features.Multivalue);
```
I suspect the problem is in TypeScript definitions of binaryen.js, but I'm not sure.
Thanks.

**Comment by dcodeIO** [comment]
May this have something to do with the `Features` enum being declared as `const`, so even though it's in a declaration file, TS then tries to inline the value into the `console.log` call? Might be testable by compiling with `--preserveConstEnums`.

**Comment by cician** [comment]
I've reproduced the problem in codesandbox: https://codesandbox.io/s/bynaryen-multivalue-node-hw2c8?file=/src/index.ts

Do you mean compiling binaryen or my project? Setting it in my project doesn't change anything and the resulting JavaScript always has the incorrect values inlined like this: ```console.log(binaryen.Features.Multivalue);``` becomes ```console.log(10 /* Multivalue */);```.

Note that index.d.ts in the binaryen.js package declares ```const enum Features``` without numeric values on its members. I'm still not very experienced with TypeScript, but unless there's a bug in the compiler I guess the declaration needs to either have them or be declared somehow differently. Removing ```const``` from ```const enum Features``` does seem to work for me and the above compiles to ```console.log(binaryen.Features.Multivalue)```. Other enums don't have this problem even though they get inlined because they aren't bitmasks.

**Comment by drew-y** [comment]
I have a fix for this in #36.

**Comment by cician** [comment]
I confirm the issue is solved in latest nightly build. Thank you.
